### PR TITLE
changelog and readme renewal, toward Oct 1 Update

### PR DESCRIPTION
**Pull request by jfkuayue**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,49 +9,36 @@ READ IN ENGLISH | [CHINESE](./README-zh.md)
 
 ## 1. General Information
 
-China Set: Trains is the train sector of the China set of OpenTTD.\
```
Comment (wensimehrp) [inline README.md:12]: Better keep that line break

File: README.md [hunk 2]
```diff
@@ -9,49 +9,36 @@ READ IN ENGLISH | [CHINESE](./README-zh.md)
...
+| Purchase cost                                                      | Change purchase cost                                                                                                                                        | 25%           |
+| Running cost                                                       | Change running cost                                                                                                                                         | 25%           |
+| Cargo age period                                                   | Change the rate cargo gets aged                                                                                                                             | Standard      |
+| Cargo age period<br />for freight wagons                           | Change the rate cargo in mail and freight wagons<br />gets aged; independent from the parameter above                                                      | Standard      |
```
Comment (wensimehrp) [inline README.md:36]: Html?

File: README.md [hunk 3]
```diff
@@ -60,14 +47,13 @@ These tools are required to build the GRF:
...
 
->[!NOTE]
->The first build will be slow since all models have to be rendered, which is a slow process.
->You can use `make -j n` to speed up the process by running multiple tasks at once where `n` is the task limit.
+> [!NOTE]
+> The first build will be slow since all models have to be rendered, which is a slow process.
+> You can use `make -j n` to speed up the process by running multiple tasks at once where `n` is the task limit.
 
-Using Linux or WSL:
+Using Linux or WSL:#!/bin/bash
```
Comment (wensimehrp) [inline README.md]: Typo? Shebang shouldn't be here

File: README.md [hunk 4]
```diff
@@ -128,8 +114,10 @@ Thanks to:
 - andythenorth
 - Belaja Lilija
 - Emperor Jake
+- dP
```
Comment (wensimehrp) [inline README.md:117]: Alphabetical order, d is before e

**Review by wensimehrp — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (jfkuayue) [reply README.md:117]: okay

File: README.md (hunk 2 above)
Comment (jfkuayue) [reply README.md:36]: yes

Comment (wensimehrp) [reply README.md:36]: remove it then. Make it a footnote or whatever thing that is markdown native.

File: README.md (hunk 4 above)
Comment (wensimehrp) [reply README.md:117]: better force push by the way; don't let the minor changes spam the commit history

**Comment by wensimehrp** [comment]
gosh you squashed many changes in one commit

File: README.md [hunk 5]
```diff
@@ -127,9 +114,11 @@ Thanks to:
 
 - andythenorth
 - Belaja Lilija
+- dP
 - Emperor Jake
 - Garlic_Bread42
 - KeepinItRail
+- peter1138
```
Comment (wensimehrp) [inline README.md:121]: I feel like you're just listing whatever people you've talked with :-1:

Comment (jfkuayue) [reply README.md:121]: ?

Comment (jfkuayue) [reply README.md:121]: No, Peter had a PR that solved extra speed for non-tilt high speed trains at sharp turns.

Comment (wensimehrp) [reply README.md:121]: whatever

**Pull request merged**